core/src/main/java/org/kiwix/kiwixmobile/core/main/ToolbarScrollingKiwixWebView.java

```diff
@@ -51,6 +51,13 @@ public ToolbarScrollingKiwixWebView(Context context, WebViewCallback callback, A
     this.toolbarView = toolbarView;
     this.bottomBarView = bottomBarView;
     this.sharedPreferenceUtil = sharedPreferenceUtil;
+    // The webview needs to be scrolled with 0 to not be slightly hidden on startup.
```

Comment:
I'd move this comment to a javadoc, also git blame provides us enough info because the commit has the ticket number in it but maybe that won't be preserved in a kotlin conversion?